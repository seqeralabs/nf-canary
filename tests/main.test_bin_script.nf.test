nextflow_process {

    name "Test Process TEST_BIN_SCRIPT"
    script "main.nf"
    process "TEST_BIN_SCRIPT"

    test("TEST_BIN_SCRIPT") {

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }

    }

}
