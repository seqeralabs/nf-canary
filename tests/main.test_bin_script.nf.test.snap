{
    "TEST_BIN_SCRIPT": {
        "content": [
            {
                "0": [
                    "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.3"
        },
        "timestamp": "2025-01-14T14:13:22.327004"
    }
}